lib/AdapterLdap.php

```diff
@@ -340,4 +340,43 @@ public function getMemberStatusByUserAndVo($user, $vo)
         }
         return Member::VALID;
     }
+
+    public function getResourceCapabilities($entityId, $userGroups)
+    {
+        $facility = $this->connector->searchForEntity(
+            $this->ldapBase,
+            '(&(objectClass=perunFacility)(entityID=' . $entityId . '))',
+            ['perunFacilityId']
+        );
+        $facilityId = $facility['perunFacilityId'][0];
```

Comment:
Extract attribute name as the class constant and replace usage of hardcoded names with this constant.